source/include/vx_reference.h

```diff
@@ -211,6 +220,18 @@ vx_uint32 ownIncrementReference(vx_reference ref, vx_enum reftype);
  */
 vx_uint32 ownDecrementReference(vx_reference ref, vx_enum reftype);
 
+/*! \brief Returns the total reference count of the object.
+ * \param [in] ref The reference to print.
+ * \ingroup group_vx_reference
+ */
+vx_uint32 ownTotalReferenceCount(vx_reference ref);
```

Comment:
I don't see this function definition anywhere, nor is it called from anywhere.

Reply:
removed not used anymore